bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiRpc.java

```diff
@@ -768,7 +768,7 @@ public void onClose(int statusCode, String description) {
 
     @Override
     public void onError(Throwable cause) {
-        logger.debug("{}: WebSocket error: {}", thingName, cause.getMessage());
+        logger.warn("{}: WebSocket error: {}", thingName, cause.getMessage());
```

Comment:
You could remove this logging completely and instead use or append `cause.getMessage()` in line 774 below. This way the cause will be included in the Thing status description, which will also be logged as well as shown in the UI.

Reply:
Thanks, if it can be shown in the UI it's even better and more immediate. I'll change the PR that way.